loadtest/user/userentity/actions.go

```diff
@@ -142,6 +142,18 @@ func (ue *UserEntity) SearchPosts(teamId, terms string, isOrSearch bool) (*model
 	return postList, nil
 }
 
+func (ue *UserEntity) GetPostsForChannel(channelId string, page, perPage int) error {
+	postlist, resp := ue.client.GetPostsForChannel(channelId, page, perPage, "")
+	if resp.Error != nil {
+		return resp.Error
+	}
+	posts := make([]*model.Post, 0, len(postlist.Posts))
+	for _, v := range postlist.Posts {
+		posts = append(posts, v)
+	}
```

Comment:
Since we've already allocated we can avoid the append and simply assign by index.

Reply:
Ok, good to know :)

Reply:
But there is not numeric index because the list is a map. You mean to include a counter and increment it instead of doing append?

Reply:
Yes, sorry for the confusion. We did something similar in https://github.com/mattermost/mattermost-load-test-ng/pull/43.

Reply:
Done